Source/Addendum/IMAUtils.swift

```diff
@@ -0,0 +1,72 @@
+/*   Copyright 2018-2019 Prebid.org, Inc.
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import Foundation
+
+@objc public enum IMAAdSlotSize: Int {
+    
+    case Size400x300
+    
+    case Size640x480
+    
+    func size () -> String {
+        switch self {
+        case .Size400x300: return "400x300"
+        case .Size640x480: return "640x480"
```

Comment:
why there are only two sizes ?

Reply:
these are the only 2 sizes IMA supports

Reply:
do you have a doc link ?

Reply:
This is in creating vast creatives in the adsManager. The same implementation can be found in android too...
<img width="849" alt="Screen Shot 2020-11-30 at 8 03 06 PM" src="https://user-images.githubusercontent.com/19392402/100683884-3f24c100-3347-11eb-971e-eaa01e501bde.png">